### PR TITLE
Notebook demo to plot shear of the E06 data

### PR DESCRIPTION
**Pull request by stephenholleran**
This is a notebook to show how to plot the shear of the E06 floating lidar dataset using the data model and the brightwind library.

As it is a tool it can be merged directly into the master branch.

### CODE REVIEW

**Comment by stephenholleran** [comment]
Hi @NikoSimisiroglou and @dnuno89,

I would be grateful if you were both able to review this and provide any comments.

Thanks very much,

**Review by dnuno89 — APPROVED** [review]
It works fine on my computer. Btw, nice plots!

**Comment by stephenholleran** [comment]
> It works fine on my computer. Btw, nice plots!

Thanks @dnuno89 .

Relating to your comment, I was thinking maybe we should have these notebooks rendered so the user doesn't have to actually get them working on their machine if they don't want to and they can still see how it should look. I think we should have them in a run state? Thought?

File: tools/Plotting the shear profile of the E06 floating lidar data utilising the WRA Data Model and the brightwind library.ipynb [hunk 1]
```diff
@@ -0,0 +1,415 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c3663f97",
+   "metadata": {},
+   "source": [
+    "Normally loading a csv into the brightwind library would be as simple as `bw.load_csv('filename.csv')`. However, due to poor formatting of the timeseries data file downloaded directly from the above link there are few extra things to do to read this in properly.\n",
+    "1. `engine='python'` is added due to the 3 leading character spaces before the 'NaN' values, Pandas throws an error as it thinks there are multiple data types, strings and numeric in the one column.\n",
+    "1. Also, due to this we need to force this to a numeric by using `.apply(pd.to_numeric, errors='coerce')`\n",
```
Comment (dnuno89) [inline tools/Plotting the shear profile of the E06 floating lidar data utilising the WRA Data Model and the brightwind library.ipynb:164]: This also forces all string columns to be NaN as they can't be converted. This only affects the last column, FLS200_S/N, which is the buoy's S/N. Maybe add a comment here commenting that, as the next cell shows all columns and displays NaNs for FLS200_S/N.

**Review by dnuno89 — COMMENTED** [review]
Just a minor comment

**Comment by dnuno89** [comment]
> > It works fine on my computer. Btw, nice plots!
> 
> Thanks @dnuno89 .
> 
> Relating to your comment, I was thinking maybe we should have these notebooks rendered so the user doesn't have to actually get them working on their machine if they don't want to and they can still see how it should look. I think we should have them in a run state? Thought?

That's a great idea, it also allows for people without Python to see it online, or avoids issues with different library versions and alike.

File: tools/Plotting the shear profile of the E06 floating lidar data utilising the WRA Data Model and the brightwind library.ipynb (hunk 1 above)
Comment (stephenholleran) [reply tools/Plotting the shear profile of the E06 floating lidar data utilising the WRA Data Model and the brightwind library.ipynb]: Thanks for pointing out. I missed that.
Done.

**Comment by stephenholleran** [comment]
I have rendered the Notebook to make it easier for people the review it without needing to pull it down and get all the Python dependencies working to run it and see the results.

**Comment by stephenholleran** [comment]
I'm going to merge this. It is only a notebook so not core schema related changes.

**Pull request merged**